src/Components/Card.js

```diff
@@ -1,25 +1,15 @@
-import React, { useContext } from "react";
```

Comment:
Refactored. Realized this logic shouldn't go here but instead should be in a container that holds the Cards.